src/config/offlineDB.connection.js

```diff
@@ -0,0 +1,10 @@
+import Dexie from 'dexie'
+
+const DB_DEFAULT_NAME = 'tknbrgoff'
```

Comment:
don't we have this as an env variable?

Reply:
i think we do, but i would rather have it harcoded as a constant somewhere